client/screens/UpdateProfile.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import PersonIcon from '@material-ui/icons/Person';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../store/users/AuthContext';
+import { FormBuilder } from '../components/FormBuilder';
+
+export function UpdateProfile() {
+  const [email, setEmail] = useState('');
```

Comment:
Maybe hydrate with user's current email

Reply:
is this what you mean? 
![image](https://user-images.githubusercontent.com/35232697/107993128-d93bf300-6f9f-11eb-9aa0-9429fcb15a2f.png)

![image](https://user-images.githubusercontent.com/35232697/107993214-e3f68800-6f9f-11eb-917f-fa8dc73b2b93.png)

Reply:
Yep! So that that way, users aren't going to need to type their email if they don't need to